feature/settings/src/main/java/com/google/jetpackcamera/settings/ui/SettingsComponents.kt

```diff
@@ -272,24 +272,35 @@ private fun getStabilizationStringRes(
  * ON - Both preview and video are stabilized.
  * HIGH_QUALITY - Video will be stabilized, preview might be stabilized, depending on the device.
  * OFF - Preview and video stabilization is disabled.
+ *
+ * @param isStabilizationSupported the enabled condition for this setting.
  */
 @Composable
 fun VideoStabilizeSetting(
     currentPreviewStabilization: Stabilization,
     currentVideoStabilization: Stabilization,
+    isStabilizationSupported: Boolean,
     setVideoStabilization: (Stabilization) -> Unit,
     setPreviewStabilization: (Stabilization) -> Unit
-
 ) {
+    Log.d(
+        "SettingsComponent",
+        "Setting component says stabilization support: $isStabilizationSupported"
+    )
     BasicPopupSetting(
         title = stringResource(R.string.video_stabilization_title),
         leadingIcon = null,
-        description = stringResource(
-            id = getStabilizationStringRes(
-                previewStabilization = currentPreviewStabilization,
-                videoStabilization = currentVideoStabilization
+        enabled = isStabilizationSupported,
+        description = when (isStabilizationSupported) {
```

Comment:
nit: instead of `when` with `true` and `false` branches here, just use an `if` statement.